docs/02.API-REFERENCE.md

```diff
@@ -6448,6 +6476,66 @@ jerry_create_arraybuffer (jerry_length_t size);
 - [jerry_value_is_arraybuffer](#jerry_value_is_arraybuffer)
 - [jerry_release_value](#jerry_release_value)
 
+# jerry_arraybuffer_set_external_free_callback
+
+**Summary**
+
+Sets the global callback which is called when the arraybuffer is no longer used.
```

Comment:
(imho) A bit of improvement " when the arraybuffer" -> "when an external arraybuffer is released / freed"